_includes/docs/pe/solution-templates/air-quality-monitoring.md

```diff
@@ -0,0 +1,74 @@
+
+* TOC 
+{:toc}
+
+{% include templates/solution-templates.md %}
```

Comment:
Don't forget to add a reference to this page from the https://thingsboard.io/use-cases/air-quality-monitoring/